### PR TITLE
add graphene to linux64/mac/win64 definitions.

### PR DESCRIPTION
**Pull request by bhearsum**
I'm in the process of adding Graphene builds, which will end up with:
graphene_larch_linux64 build
graphene_larch_macosx64 build
graphene_larch_win64 build

...and debug versions. I'm a bit overwhelmed by this file and not sure whether I need to update any data structures that are further down. I noticed that every single b2g device has its own definitions, but things such as Firefox nightlies appear to be caught in wider globs. I've got a patch for treeherder-ui coming too, but please let me know if there's anything else I should update here!

### CODE REVIEW

File: treeherder/etl/buildbot.py [hunk 1]
```diff
@@ -59,6 +59,14 @@
         }
     },
     {
+        'regex': re.compile(r'^graphene.*_linux64', re.IGNORECASE),
```
Comment (edmorley) [inline treeherder/etl/buildbot.py]: I believe this needs to be higher up, since the regex above will win (they are executed in order listed here).

**Comment by edmorley** [comment]
I don't suppose you could add a test? Use this mulet one as an example:
https://github.com/mozilla/treeherder-service/blob/master/tests/etl/test_buildbot.py#L506

**Comment by bhearsum** [comment]
Anything else I need to address before this is merged? No rush, just unsure of the current state.

**Comment by edmorley** [comment]
No, this looks fine now - thank you :-)
(Github doesn't notify on new commits, only comments, so I hadn't seen it had been updated)

**Pull request merged**